pkg/routes/routes.go

```diff
@@ -59,7 +60,16 @@ func (r *DMSRouter) Register(c echo.Context) error {
 
 // Status is an internal endpoint
 func (r *DMSRouter) Status(c echo.Context) error {
-	all, err := r.Store.All()
+	all, err := r.Store.AllCheckins()
+	if err != nil {
+		return c.String(http.StatusInternalServerError, err.Error())
+	}
+	return c.JSON(http.StatusOK, all)
+}
+
+// Incidents returns a list of recent incidents
+func (r *DMSRouter) Incidents(c echo.Context) error {
```

Comment:
What's the difference between this endpoint and `Status`?

Reply:
Incidents will show a history of failed checkins, not the live status